### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by durgamanidevarapu**
please do code review.. as week1 changes are already in master..

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +49,33 @@
...
+			<version>2.9.2</version>
+		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>2.9.2</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
```
Comment (ashoksuryap) [inline pom.xml:64]: Do we need this?

File: src/main/java/com/product/controller/ProductServiceController.java [hunk 2]
```diff
@@ -1,101 +1,146 @@
...
+
+    private Product updateProductEntity(ProductDto productDto, Product product) {
+
+        Product productEntity = convertToEntity(productDto);
+        productEntity.setId(product.getId());
+        if (StringUtils.isEmpty(productEntity.getName())) {
+            productEntity.setName(product.getName());
+        }
+        if (StringUtils.isEmpty(product.getDescription())) {
+            productEntity.setName(product.getDescription());
```
Comment (ashoksuryap) [inline src/main/java/com/product/controller/ProductServiceController.java]: Need to update properties of DB entity from dto. you are doing it opposite .please check

File: pom.xml [hunk 3]
```diff
@@ -47,6 +49,33 @@
...
+		<dependency>
+			<groupId>org.modelmapper</groupId>
+			<artifactId>modelmapper</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+		<!--<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<scope>runtime</scope>
+		</dependency>-->
```
Comment (ashoksuryap) [inline pom.xml]: why this is commented?

File: pom.xml (hunk 1 above)
Comment (durgamanidevarapu) [reply pom.xml:64]: I am using this in test class for content

File: src/main/java/com/product/controller/ProductServiceController.java (hunk 2 above)
Comment (durgamanidevarapu) [reply src/main/java/com/product/controller/ProductServiceController.java]: I have changed the code and committed ..

File: pom.xml (hunk 3 above)
Comment (durgamanidevarapu) [reply pom.xml]: I faced some issue previously ..now i added it and committed

**Pull request closed without merging**